_sass/_layout.scss

```diff
@@ -243,3 +243,34 @@
 .pagination {
   text-align: center;
 }
+
+// Social Media Biography
+
+.bio-social-media {
+  ul {
```

Comment:
I meant changing the `ul` tag selector to a class like `.boi-social-media-list`. The new class would be added to the `ul`.